testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/management/persistence/yaml/YamlExtensionTestCase.java

```diff
@@ -330,7 +331,7 @@ public void testSimpleYamlWithCliBootOps() throws Exception {
 
     @Test
     public void testYamlChangesAppliedInAdminOnlyModeWithoutBootCliScript() throws Exception {
-        container.start(null, null, Server.StartMode.ADMIN_ONLY, System.out, false, null, null, null, null, new Path[]{testYaml});
+        container.start(null, null, Server.StartMode.NORMAL, System.out, false, null, null, null, null, new Path[]{testYaml});
```

Comment:
This is contrary to what the test method name says is being tested.

Reply:
Upss, so evident. Fixed, move it back to ADMIN_ONLY to make honor test intention.